test/modules/diffusion/diffusion_2d_2_unstructured_ip.lua

```diff
@@ -1,105 +0,0 @@
--- 2D Diffusion test with Dirichlet BCs.
--- SDM: PWLD
--- Test: Max-value=0.29685
-num_procs = 4
-
-
-
-
-
---############################################### Check num_procs
-if (check_num_procs==nil and number_of_processes ~= num_procs) then
-    Log(LOG_0ERROR,"Incorrect amount of processors. " ..
-                      "Expected "..tostring(num_procs)..
-                      ". Pass check_num_procs=false to override if possible.")
-    os.exit(false)
-end
-
---############################################### Setup mesh
-meshgen1 = mesh.MeshGenerator.Create
-({
-  inputs =
-  {
-    mesh.FromFileMeshGenerator.Create
-    ({
-      filename = "../../../resources/TestMeshes/TriangleMesh2x2.obj"
-    })
-  }
-})
-mesh.MeshGenerator.Execute(meshgen1)
-
---############################################### Set Material IDs
-vol0 = mesh.RPPLogicalVolume.Create({infx=true, infy=true, infz=true})
-VolumeMesherSetProperty(MATID_FROMLOGICAL,vol0,0)
-VolumeMesherSetupOrthogonalBoundaries()
-
-
---############################################### Add materials
-materials = {}
-materials[0] = PhysicsAddMaterial("Test Material");
-
-PhysicsMaterialAddProperty(materials[0],SCALAR_VALUE)
-PhysicsMaterialSetProperty(materials[0],SCALAR_VALUE,SINGLE_VALUE,1.0)
-
---############################################### Setup Physics
-phys1 = DiffusionCreateSolver()
-SolverSetBasicOption(phys1,"discretization_method","PWLD_MIP")
-SolverSetBasicOption(phys1,"residual_tolerance",1.0e-6)
-
---############################################### Set boundary conditions
---DiffusionSetProperty(phys1,"boundary_type",0,"reflecting",1.0)
---DiffusionSetProperty(phys1,"boundary_type",1,"vacuum",2.0)
---DiffusionSetProperty(phys1,"boundary_type",2,"reflecting",3.0)
---DiffusionSetProperty(phys1,"boundary_type",3,"vacuum",4.0)
-
---############################################### Initialize and Execute Solver
-DiffusionInitialize(phys1)
-DiffusionExecute(phys1)
-
---############################################### Get field functions
-fftemp,count = SolverGetFieldFunctionList(phys1)
-
---############################################### Slice plot
-slice2 = FFInterpolationCreate(SLICE)
-FFInterpolationSetProperty(slice2,SLICE_POINT,0.0,0.0,0.025)
-FFInterpolationSetProperty(slice2,ADD_FIELDFUNCTION,fftemp[1])
-
-FFInterpolationInitialize(slice2)
-FFInterpolationExecute(slice2)
-
---############################################### Line plot
-line0 = FFInterpolationCreate(LINE)
-FFInterpolationSetProperty(line0,LINE_FIRSTPOINT,-1.0,0.01,0.0)
-FFInterpolationSetProperty(line0,LINE_SECONDPOINT, 1.0,0.01,0.0)
-FFInterpolationSetProperty(line0,LINE_NUMBEROFPOINTS, 100)
-FFInterpolationSetProperty(line0,ADD_FIELDFUNCTION,fftemp[1])
```

Comment:
let's make sure we have existing tests that still exercise these 2 options, be it just for the syntax, otherwise I am afraid we will start to forget about that output possibility.

Reply:
I checked and this is exercised at least in `c_diffusion_2d_1a_linear.lua` and `d_diffusion_2d_1a_linear.lua`.